### PR TITLE
[id] Clean up various broken links

### PR DESCRIPTION
**Pull request by tallclair**
Clean up broken links reported through the HackerOne bug bounty program.

Localized version of #32783

/committee security-response

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Are these changes already made upstream?

**Comment by tallclair** [comment]
by upstream do you mean the english content?

I deleted the broken addon in https://github.com/kubernetes/website/commit/b0e4e7a03c140abc2b8925ac84a4f15beb732d08, but found that it had moved when I sent this update. I can add it back with the updated link to the english if you'd like, or we can remove it here.

**Comment by tallclair** [comment]
I went ahead and opened a PR to add it back, here: https://github.com/kubernetes/website/pull/32938

File: content/id/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ Ikuti bagian "With Linux Bridge devices" dari [tutorial yang sangat bagus ini](h
...
+[Multus](https://github.com/k8snetworkplumbingwg/multus-cni) adalah plugin Multi CNI untuk mendukung fitur Banyak Jaringan di Kubernetes menggunakan objek jaringan berbasis CRD di Kubernetes.
 
 Multus mendukung semua [plugin referensi](https://github.com/containernetworking/plugins) (mis. [Flannel](https://github.com/containernetworking/plugins/tree/master/plugins/meta/flannel), [DHCP](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/dhcp), [Macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main / macvlan)) yang mengimplementasikan spesifikasi CNI dan plugin pihak ke-3 (mis. [Calico](https://github.com/projectcalico/cni-plugin), [Weave](https://github.com/weaveworks/weave ), [Cilium](https://github.com/cilium/cilium), [Contiv](https://github.com/contiv/netplugin)). Selain itu, Multus mendukung [SRIOV](https://github.com/hustcat/sriov-cni), [DPDK](https://github.com/Intel-Corp/sriov-cni), [OVS- DPDK & VPP](https://github.com/intel/vhost-user-net-plugin) beban kerja di Kubernetes dengan aplikasi cloud asli dan aplikasi berbasis NFV di Kubernetes.
```
Comment (jihoon-seo) [inline content/id/docs/concepts/cluster-administration/networking.md:184]: - Currently in upstream (en):

https://github.com/kubernetes/website/blob/89c91ff2c5a78ffb9b6881079726b43436b6ff1a/content/en/docs/concepts/cluster-administration/networking.md?plain=1#L174

```suggestion
Multus mendukung semua [plugin referensi](https://github.com/containernetworking/plugins) (mis. [Flannel](https://github.com/containernetworking/cni.dev/blob/main/content/plugins/v0.9/meta/flannel.md), [DHCP](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/dhcp), [Macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan)) yang mengimplementasikan spesifikasi CNI dan plugin pihak ke-3 (mis. [Calico](https://github.com/projectcalico/cni-plugin), [Weave](https://github.com/weaveworks/weave), [Cilium](https://github.com/cilium/cilium), [Contiv](https://github.com/contiv/netplugin)). Selain itu, Multus mendukung [SRIOV](https://github.com/hustcat/sriov-cni), [DPDK](https://github.com/k8snetworkplumbingwg/sriov-cni), [OVS- DPDK & VPP](https://github.com/intel/userspace-cni-network-plugin) beban kerja di Kubernetes dengan aplikasi cloud asli dan aplikasi berbasis NFV di Kubernetes.
```

File: content/id/docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ Plugin ini dirancang untuk secara langsung mengkonfigurasi dan _deploy_ dalam VP
 
 ### Contiv
 
-[Contiv](https://github.com/contiv/netplugin) menyediakan jaringan yang dapat dikonfigurasi (_native_ l3 menggunakan BGP, _overlay_ menggunakan vxlan, classic l2, atau Cisco-SDN / ACI) untuk berbagai kasus penggunaan. [Contiv](http://contiv.io) semuanya open sourced.
+[Contiv](https://github.com/contiv/netplugin) menyediakan jaringan yang dapat dikonfigurasi (_native_ l3 menggunakan BGP, _overlay_ menggunakan vxlan, classic l2, atau Cisco-SDN / ACI) untuk berbagai kasus penggunaan. [Contiv](https://contivpp.io/) semuanya open sourced.
 
 ### Contrail / Tungsten Fabric
```
Comment (jihoon-seo) [inline content/id/docs/concepts/cluster-administration/networking.md:111]: Could you please update the markdown hyperlinks to work?

L116
```diff
-[DANM] (https://github.com/nokia/danm) adalah solusi jaringan untuk beban kerja telco yang berjalan di kluster Kubernetes. Dibangun dari komponen-komponen berikut:
+[DANM](https://github.com/nokia/danm) adalah solusi jaringan untuk beban kerja telco yang berjalan di kluster Kubernetes. Dibangun dari komponen-komponen berikut:
```

L128
```diff
-[Flannel] (https://github.com/coreos/flannel#flannel) adalah jaringan overlay yang sangat sederhana yang memenuhi persyaratan Kubernetes. Banyak orang telah melaporkan kesuksesan dengan Flannel dan Kubernetes.
+[Flannel](https://github.com/coreos/flannel#flannel) adalah jaringan overlay yang sangat sederhana yang memenuhi persyaratan Kubernetes. Banyak orang telah melaporkan kesuksesan dengan Flannel dan Kubernetes.
```

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 3]
```diff
@@ -25,14 +25,14 @@ Laman ini akan menjabarkan beberapa *add-ons* yang tersedia serta tautan instruk
 * [Canal](https://github.com/tigera/canal/tree/master/k8s-install) menggabungkan Flannel dan Calico, menyediakan jaringan serta *policy* jaringan.
 * [Cilium](https://github.com/cilium/cilium) merupakan *plugin* jaringan L3 dan *policy* jaringan yang dapat menjalankan *policy* HTTP/API/L7 secara transparan. Mendukung mode *routing* maupun *overlay/encapsulation*.
 * [CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie) memungkinkan Kubernetes agar dapat terkoneksi dengan beragam *plugin* CNI, seperti Calico, Canal, Flannel, Romana, atau Weave dengan mulus.
```
Comment (jihoon-seo) [inline content/id/docs/concepts/cluster-administration/addons.md:27]: - Updating the URL of `CNI-Genie`
- #32937 is removing `Romana` from example list.

```suggestion
* [CNI-Genie](https://github.com/cni-genie/CNI-Genie) memungkinkan Kubernetes agar dapat terkoneksi dengan beragam *plugin* CNI, seperti Calico, Canal, Flannel, atau Weave dengan mulus.
```

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 4]
```diff
@@ -25,14 +25,14 @@ Laman ini akan menjabarkan beberapa *add-ons* yang tersedia serta tautan instruk
...
 * [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) merupakan platform SDN yang menyediakan *policy-based* jaringan antara Kubernetes Pods dan non-Kubernetes *environment* dengan *monitoring* visibilitas dan keamanan.
-* [Romana](http://romana.io) merupakan solusi jaringan  *Layer* 3 untuk jaringan pod yang juga mendukung [*NetworkPolicy* API](/id/docs/concepts/services-networking/network-policies/). Instalasi Kubeadm *add-on* ini tersedia [di sini](https://github.com/romana/romana/tree/master/containerize).
+* [Romana](https://github.com/romana/romana) merupakan solusi jaringan  *Layer* 3 untuk jaringan pod yang juga mendukung [*NetworkPolicy* API](/id/docs/concepts/services-networking/network-policies/). Instalasi Kubeadm *add-on* ini tersedia [di sini](https://github.com/romana/romana/tree/master/containerize).
```
Comment (jihoon-seo) [inline content/id/docs/concepts/cluster-administration/addons.md:35]: - Currently in upstream (en):

https://github.com/kubernetes/website/blob/89c91ff2c5a78ffb9b6881079726b43436b6ff1a/content/en/docs/concepts/cluster-administration/addons.md?plain=1#L33

- #32937 is updating this line on current upstream (en).

```suggestion
* [Romana](https://github.com/romana/romana) merupakan solusi jaringan  *Layer* 3 untuk jaringan pod yang juga mendukung [*NetworkPolicy*](/id/docs/concepts/services-networking/network-policies/) API.
```

**Review by jihoon-seo — COMMENTED** [review]
@tallclair PTAL!

**Comment by Sea-n** [comment]
Hey @tallclair,

Thanks for purposing this PR. But since you didn't address the comments and stale for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.

**Pull request closed without merging**